code-studio/src/js/components/Grid.jsx

```diff
@@ -0,0 +1,100 @@
+/* global React */
+
+var mazeUtils = require('@cdo/apps/maze/mazeUtils');
+
+var Cell = React.createClass({
+  propTypes: {
+    cell: React.PropTypes.object.isRequired,
+    row: React.PropTypes.number.isRequired,
+    col: React.PropTypes.number.isRequired,
+    selected: React.PropTypes.bool.isRequired,
+    onSelectionChange: React.PropTypes.func.isRequired,
+    skin: React.PropTypes.string.isRequired,
+  },
+
+  clickCell: function (event) {
```

Comment:
might call his handleClickCell or onClickCell for consistency?

I think that maybe our pattern is that the function name is handle* but we pass it as on*? I don't think we're super consistent about that though.